controllers/disruption_controller.go

```diff
@@ -431,6 +434,40 @@ func (r *DisruptionReconciler) cleanDisruption(instance *chaosv1beta1.Disruption
 	return cleaned, nil
 }
 
+func (r *DisruptionReconciler) handleOrphanedChaosPods(req ctrl.Request) error {
+	ls := make(map[string]string)
+
+	ls[chaostypes.DisruptionNameLabel] = req.Name
+	ls[chaostypes.DisruptionNamespaceLabel] = req.Namespace
+
+	chaosPods, err := r.getChaosPods(nil, ls)
+	if err != nil {
+		return err
+	}
+
+	for _, chaosPod := range chaosPods {
+		r.handleMetricSinkError(r.MetricsSink.MetricOrphanFound([]string{"name:" + req.Name, "namespace:" + req.Namespace}))
+
+		r.log.Warnw("orphaned chaos pod detected, will attempt to delete", "chaosPod", chaosPod.Name)
```

Comment:
Error log gives a stack trace which is unhelpful here. We can rely on the metric to detect issues, rather than needing this to be error instead of warn